### PR TITLE
Maintain siblings order on CSV import, refs #13414

### PR DESCRIPTION
**Pull request by jraddaoui**
Maintain siblings order when the nested set update is disabled.
Add a placeholder lft value incrementing the max lft value from
the resource's siblings. This placeholder will overlap with the
parent's rgt value until the nested set is rebuilt.

### CODE REVIEW

File: lib/model/QubitInformationObject.php [hunk 1]
```diff
@@ -225,6 +225,50 @@ public function save($connection = null)
...
+          
+        $connection->exec($sql);
+
+        $connection->commit();
+      }
+      catch (Exception $e)
+      {
+        $connection->rollBack();
+
+        throw $e;
```
Comment (sbreker) [inline lib/model/QubitInformationObject.php]: What state will io be in if this rollback and exception are thrown? parent::save() is called before, but all child records etc are called after. Will the io be saved, but all child recs not be created?

**Review by sbreker — COMMENTED** [review]
Added one question regarding save() behaviour if the exception is thrown

File: lib/model/QubitInformationObject.php (hunk 1 above)
Comment (jraddaoui) [reply lib/model/QubitInformationObject.php]: @sbreker, that's a great point, thanks!

I'll move down in the method, just before indexing.

**Review by sbreker — APPROVED** [review]
Thanks for making that change @jraddaoui !  Looks good

**Pull request merged**